### PR TITLE
Add option to delete the previous comment and create a new one instead of editing

### PR DESCRIPTION
**Pull request by jakemhiller**
In our pull requests having lots of comments and changes can make it hard to find the PR comment from this action.

I figured the best solution would be to add an option to replace the previous comment with a new comment instead of editing the old comment, so the comment shows up at the bottom of the PR.

Changes:

1. added the `replace` option to the action definition
1. added a `deleteComment` method to remove the previous comment if it exists
1. updated `createComment` to take the `previousBody` argument. Which is used if `replace` and `append` are both true.
1. added a basic test for `deleteComment`

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -8,6 +8,9 @@ inputs:
   append:
     description: "Indicate if new comment messages should be appended to previous comment message"
     required: false
+  replace:
+    description: "Indicate if previous comment should be removed before creating a new comment"
+    required: false
```
Comment (marocchino) [inline action.yml:13]: `recreate` seems to be a better name for this behavior, since default behavior also replaces inside of a comment.

Comment (jakemhiller) [reply action.yml]: yeah, that's better. just updated.

**Comment by marocchino** [comment]
Thanks!

**Comment by jakemhiller** [comment]
@marocchino just noticed this was closed, is it going to end up in master at some point?

**Comment by marocchino** [comment]
I thought it was merged;

**Comment by marocchino** [comment]
I taged master as `v1.5.0` and `v1`. enjoy it. 😅

**Pull request merged**